src/main/java/com/vlkan/maven/plugins/quasar/QuasarInstrumentorMojo.java

```diff
@@ -83,9 +83,6 @@
     public void execute() throws MojoExecutionException {
         getLog().info("Instrumenting Quasar classes...");
 
-        if (buildDirectory == null || !buildDirectory.isDirectory())
```

Comment:
buildDirectory is normally never null, because it points ${project.build.directory} maven default property which uses the parent's directory of the pom